modules/migration-about-configuring-proxies.adoc

```diff
@@ -9,14 +9,14 @@
 [id="migration-about-configuring-proxies_{context}"]
 = Proxy configuration
 
-For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-full} Operator because these versions do not support a cluster-wide `proxy` object.
+For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-first} Operator because these versions do not support a cluster-wide `proxy` object.
```

Comment:
```suggestion
For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-short} Operator because these versions do not support a cluster-wide `proxy` object.
```